src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -217,6 +217,12 @@ def __post_init__(self):
                 f"Given platform constraints {self.platforms} for internal only pex request: "
                 f"{self}."
             )
+        if self.internal_only and self.complete_platforms:
```

Comment:
We check this for platforms above, but neglected to check it for complete_platforms.